egs/iban/s5/local/nnet3/run_tdnn.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# this is the standard "tdnn" system, built in nnet3; it's what we use to
```

Comment:
I recommend developers to add the name of the recipe/script from which the current script was adapted. This helps understand some choices in case we are trying to tune things in the future.